### PR TITLE
refactor: remove Composer autoloading of auth helper

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
See https://forum.codeigniter.com/showthread.php?tid=88545 and #872

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: tests/_support/TestCase.php [hunk 1]
```diff
@@ -26,6 +26,9 @@ protected function setUp(): void
         $settings                 = new Settings($configSettings);
         Services::injectMock('settings', $settings);
 
+        // Load helpers that should be autoloaded
+        helper(['auth', 'setting']);
+
         // Ensure from email is available anywhere during Tests
         helper('setting');
```
Comment (datamweb) [inline tests/_support/TestCase.php:33]:
```suggestion
```

Comment (kenjis) [reply tests/_support/TestCase.php]: Thanks!

**Comment by datamweb** [comment]
Unittest /php7. 4 

https://docs.orcle.com/cd/E11882_01/server.112/e16604/ch_thirteen.htm#SQPUG136

Screenshot_2023-10-11-04-54-35-451-edit_com android chrome

File: src/Commands/Setup.php [hunk 2]
```diff
@@ -236,24 +237,61 @@ private function replace(string $file, array $replaces): bool
...
+
+        $path      = $this->distPath . $file;
+        $cleanPath = clean_path($path);
+
+        if (! is_file($path)) {
+            $this->error("  Not found file '{$cleanPath}'.");
+
+            return;
+        }
+
```
[the remark this answers is not in the document]
Comment (kenjis) [reply src/Commands/Setup.php]: Good catch! Removed.

**Comment by datamweb** [comment]
@kenjis we have used `helper('setting');` in several places, can you remove?

**Comment by kenjis** [comment]
Testing with OCI8 always fails, but I don't know why.

```
Run echo -e "ALTER SESSION SET CONTAINER = XEPDB1;\nCREATE BIGFILE TABLESPACE \"TEST\" DATAFILE '/opt/oracle/product/18c/dbhomeXE/dbs/TEST' SIZE 10M AUTOEXTEND ON MAXSIZE UNLIMITED SEGMENT SPACE MANAGEMENT AUTO EXTENT MANAGEMENT LOCAL AUTOALLOCATE;\nCREATE USER \"ORACLE\" IDENTIFIED BY \"ORACLE\" DEFAULT TABLESPACE \"TEST\" TEMPORARY TABLESPACE TEMP QUOTA UNLIMITED ON \"TEST\";\nGRANT CONNECT,RESOURCE TO \"ORACLE\";\nexit;" | /lib/oracle/18.5/client64/bin/sqlplus -s sys/Oracle18@localhost:1521/XE as sysdba
ERROR:
ORA-12514: TNS:listener does not currently know of service requested in connect
descriptor


SP2-0306: Invalid option.
Usage: CONN[ECT] [{logon|/|proxy} [AS {SYSDBA|SYSOPER|SYSASM|SYSBACKUP|SYSDG|SYSKM|SYSRAC}] [edition=value]]
where <logon> ::= <username>[/<password>][@<connect_identifier>]
      <proxy> ::= <proxyuser>[<username>][/<password>][@<connect_identifier>]
SP2-0306: Invalid option.
Usage: CONN[ECT] [{logon|/|proxy} [AS {SYSDBA|SYSOPER|SYSASM|SYSBACKUP|SYSDG|SYSKM|SYSRAC}] [edition=value]]
where <logon> ::= <username>[/<password>][@<connect_identifier>]
      <proxy> ::= <proxyuser>[<username>][/<password>][@<connect_identifier>]
SP2-0157: unable to CONNECT to ORACLE after 3 attempts, exiting SQL*Plus
Error: Process completed with exit code 1.
```
https://github.com/codeigniter4/shield/actions/runs/6489501416/job/17623872761?pr=893

**Comment by kenjis** [comment]
Removed `helper('setting')`.

**Review by datamweb — APPROVED** [review]
LGTM!
Thank you!

**Comment by datamweb** [comment]
> Testing with OCI8 always fails, but I don't know why.

This issue is not related to this PR.
Below test failure was observed even before this PR.

https://github.com/datamweb/shield/actions/runs/6498102065/job/17648716049

**Comment by datamweb** [comment]
All checks have passed!

**Pull request merged**